README.md

```diff
@@ -58,6 +58,14 @@ Example `~/.aws/config` :
      # Example configuration
      export AWS_PROFILE="engineering"
  ```
+#### 3) Proxy
+
+Set your http_proxy in a `.envrc` file or export it in your shell. (See example [.envrc file](.envrc_example))
+
+ ```bash
+   export http_proxy="http://proxy.com:123"
```

Comment:
in code you are referring to this config as `aws_http_proxy`.. I guess this is a typo here than!